docs/pages/includes/teleport-connect-telemetry.mdx

```diff
@@ -0,0 +1,18 @@
+When you first start the app, Teleport Connect asks for permission to collect & send telemetry data.
+This includes tracking events such as:
+- Login
+- Starting an SSH, database, or kube session
+- File transfer during an SSH session
+- Managing access requests
+- Asking about job role
+
+On login, we also collect some device-related data:
+
+- Operating system and its version
+- App version
+- Processor architecture
+
+The full list of events and collected data is defined as [protocol buffer messages in connect.proto](https://github.com/gravitational/teleport/tree/branch/v(=teleport.major_version=)/proto/prehog/v1alpha/connect.proto).
```

Comment:
Looks like this PR should be blocked from merging until #20810 is backported into a release branch, as this link does not yet work with any version branch.

Reply:
Correct, I was sure it was backported.

Reply:
Looks like it's been backported 👍